mod_reforged/hooks/config/character.nut

```diff
@@ -36,3 +36,11 @@ local getClone = ::Const.CharacterProperties.getClone;
 ::Const.ProjectileSprite.push("rf_projectile_flaming_arrow");
 
 ::Const.Movement.AutoEndTurnBelowAP = 1;
+
+::Const.CharacterProperties.getSurroundedBonus <- function( _targetEntity )
```

Comment:
This should be a function in actor, not properties. Then in the function in the actor, you use the values from the character's CurrentProperties.